ReverseWordsInAString.java

```diff
@@ -0,0 +1,24 @@
+class ReverseWordsInAString {
+    public String reverseWords(String s) {
+        // remove leading spaces
+        s = s.trim();
+        // split by multiple spaces
+        List<String> wordList = Arrays.asList(s.split("\\s+"));

```

Comment:
It may be nice to drop a comment that `\s` matches any whitespace character, for the reviewer.  I had to look it up, at least.  I like that you're using RegEx here to simplify your implementation -- there's no need for you to write the string splitter!